From ff8c8b16649e22a8db99f59e1c590f37ce23df5a Mon Sep 17 00:00:00 2001
From: Demetris Ierokipides <ierokipides.dem@gmail.com>
Date: Thu, 22 Dec 2022 04:55:50 +0200
Subject: [PATCH] rk3288: add GPU 500 Mhz back Undoing the very bizarre
 mainline kernel patch, 75481833c6dbab4c29d15452f6b4337c16f5407b which main
 purpose is to sync some 3.14 kernels hacks to mainline kernels, for reasons
 that only matter for a few Chromebooks, and shove it down the throat of every
 RK3288 user.

---
 arch/arm/boot/dts/rk3288.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index 487b0e03d..a052198f5 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -1298,6 +1298,10 @@ opp-400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <1100000>;
 		};
+		opp-500000000 {
+			opp-hz = /bits/ 64 <500000000>;
+			opp-microvolt = <1200000>;
+		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <1250000>;
-- 
2.34.1

